packages/fluentui/react-northstar/test/specs/utils/felaRenderer-test.tsx

```diff
@@ -1,16 +1,20 @@
-import * as React from 'react';
+import { createFelaRenderer } from '@fluentui/react-northstar-fela-renderer';
+import { emptyTheme } from '@fluentui/styles';
 import { createSnapshot } from 'jest-react-fela';
-import { EmptyThemeProvider } from 'test/utils';
-import Box from 'src/components/Box/Box';
+import * as React from 'react';
+
 import Animation from 'src/components/Animation/Animation';
-import Provider from 'src/components/Provider/Provider';
+import Box from 'src/components/Box/Box';
 import Text from 'src/components/Text/Text';
-import { felaRenderer } from 'src/utils';
+import { EmptyThemeProvider } from 'test/utils';
+
+const fluentRenderer = createFelaRenderer();
```

Comment:
Should we rename this to `fluentRenderer-test.tsx`?

Reply:
I don't think so as it still tightly coupled to Fela. And it's impossible to move it `renderer` package as it's coupled with our components.

Reply:
Agreed to move out this test to a `renderer` package.

Reply:
Fixed in 84a5987